packages/core/integration-tests/test/cache.js

```diff
@@ -3971,7 +3971,7 @@ describe('cache', function () {
               let pluginContents = await overlayFS.readFile(plugin, 'utf8');
               await overlayFS.writeFile(
                 plugin,
-                pluginContents.replace('green', 'blue'),
```

Comment:
`@parcel/css` prints this as `#00f`, so had to change the color to something that prints the same in both transformers